src/azul/chalice.py

```diff
@@ -404,11 +407,23 @@ def _log_request(self):
                 'query': self.current_request.query_params,
                 'headers': self.current_request.headers
             }
-            log.info('Received %s request for %r, with %s.',
+            log.info('Received %s request for %r, with %s',
                      context['httpMethod'],
                      context['path'],
                      json.dumps(request_info, cls=self._LogJSONEncoder))
 
+            body = self.current_request.json_body
+            if not body:
```

Comment:
Isn't this conflating None and the empty string? And for responses, don't we log the fact that the body is absent?

Reply:
>Isn't this conflating None and the empty string?

My bad, it skipped me, ty.

>And for responses, don't we log the fact that the body is absent?

Yeah, the `http_body_log_message` gracefully handles that aspect:

https://github.com/DataBiosphere/azul/blob/ccc785feca3ec343b8ab79ef9342f3520f659b30/src/azul/logging.py#L168-L175

I've renamed the `msg` variable to `body_len_msg`, with the intent of making the purpose of the variable more evident.